spec/support/go_mod.rb

```diff
@@ -0,0 +1,42 @@
+class GoMod
+  attr_reader :path, :go_version, :requires
+
+  def initialize(path)
+    require 'pathname'
+    @path     = Pathname.new(path)
+    @requires = {}
+  end
+
+  def parse
+    return if @parsed
+    parse_block = nil
+
+    path.read.each_line do |line|
+      if line.start_with?("go ")
+        parse_go_version(line)
+        next
+      elsif line.start_with?("require (")
```

Comment:
Is the space required here? Should that be an optional space?

Reply:
Removed as discussed